src/main/java/org/apache/commons/compress/archivers/tar/TarArchiveInputStream.java

```diff
@@ -516,6 +713,22 @@ private void paxHeaders() throws IOException{
                                 final String value = new String(rest, 0,
                                                           restLen - 1, CharsetNames.UTF_8);
                                 headers.put(keyword, value);
+
+                                // for 0.0 PAX Headers
+                                if(keyword.equals("GNU.sparse.offset")) {
+                                    sparseHeader = new TarArchiveStructSparse(Long.parseLong(value), 0);
```

Comment:
better extract the keys into a static String

Reply:
Agree. There're some other keys exist in `TarArchiveEntry`, like this:

https://github.com/apache/commons-compress/blob/master/src/main/java/org/apache/commons/compress/archivers/tar/TarArchiveEntry.java#L1126-L1129

I'm wondering if a refactor is needed or not for the existing code.